sdk/eventhub/event-hubs/src/diagnostics/messageSpan.ts

```diff
@@ -25,3 +28,22 @@ export function createMessageSpan(
 
   return span;
 }
+
+/** @internal */
+export function createContextForParentSpan(parentSpan: SpanContext | Span | null | undefined): Context {
+  if (isSpan(parentSpan)) {
+    return setSpan(otContext.active(), parentSpan);
```

Comment:
again I'm really curious what otContext.active() returns when we're in the noop case